tests/FileManipulation/MissingReturnTypeTest.php

```diff
@@ -533,9 +533,9 @@ function bar() {
                     }',
                 'output' => '<?php
                     /**
-                     * @return null|string
+                     * @return string|void
```

Comment:
`void` in a union looks wrong. Looking at the code here I'd expect `null|string` actually.